src/backend/distributed/sql/udfs/citus_internal_add_object_metadata/11.0-1.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_internal_add_object_metadata(
+							typeText text,
```

Comment:
Marking this function as STRICT actually avoids lots of trouble. Why do we allow NULL values for `distribution_argument_index` and `colocationid`?

I think this is again relevant to the other `single integer allocation` discussion. If we pass InvalidColocationId and InvalidDistArgument instead of `NULL`, can't we mark this function as strict?

I'd rather avoid this whole pointer idea for single integers and mark this as STRICT